build/dependencies.yaml

```diff
@@ -102,15 +102,15 @@ dependencies:
   # This entry is a stub of the major version to allow dependency checks to
   # pass when building Kubernetes using a pre-release of Golang.
   - name: "golang: 1.<major>"
-    version: 1.18
+    version: 1.19.4
```

Comment:
This should be `1.<major>`, i.e. `1.19`. See on master for example: https://github.com/kubernetes/kubernetes/blob/7814d10882d5f159630ccbbaf17c3e014d4f7835/build/dependencies.yaml#L105

Reply:
As discussed on Slack, it will be fixed in a follow-up PR along with the comment from @aojea.

Ref: https://kubernetes.slack.com/archives/C2C40FMNF/p1671026815368429

Reply:
opened https://github.com/kubernetes/kubernetes/pull/114616 with this follow-up